utils/getSubscription.py

```diff
@@ -0,0 +1,82 @@
+from categories.models import Category,UserSubscription
+
+class getSubscription:
+    should_subscribe = {}
```

Comment:
Remove these class variables

Reply:
they are used i thought it was better to declare them as class variable rather than assing them as function parameters